Automattic/jetpack
# Mirroring: Mirror release branches, conditionally

#### Changes proposed in this Pull Request:

Release branches are of the form "*/branch-*", e.g. "jetpack/branch-9.4".
    
These probably only make sense for one specific project that's being released, so only mirror them to a project if the project's composer.json declares a matching prefix.

Then set up a prefix for Jetpack.

#### Jetpack product discussion


p1610990021274000-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Locally run `.github/files/build-all-projects.sh`
* Locally run `.github/files/push-all-projects.sh` with environment variable `GITHUB_REF` set to something like "refs/heads/jetpack/branch-9.4". See that skips wanting to push to everything except Automattic/jetpack-production.
* Repeat the above two steps, but with `GITHUB_REF` set to something like "refs/heads/notjetpack/branch-9.4". See that it skips wanting to push anything.

#### Proposed changelog entry for your changes:


* None needed.
